tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/KafkaTests.cs

```diff
@@ -71,7 +71,7 @@ public void SubmitsTraces(string packageVersion, string metadataSchemaVersion)
             using var agent = EnvironmentHelper.GetMockAgent();
             using var processResult = RunSampleAndWaitForExit(agent, arguments: topic, packageVersion: packageVersion);
 
-            var allSpans = agent.WaitForSpans(TotalExpectedSpanCount, timeoutInMilliseconds: 10_000);
+            var allSpans = agent.WaitForSpans(TotalExpectedSpanCount, timeoutInMilliseconds: 60_000);
```

Comment:
We shouldn't need to wait 60s here... the process has already exited at this point, so this mostly just slows down CI in the case of failures...